src/eredis.app.src

```diff
@@ -1,6 +1,6 @@
 {application,eredis,
              [{description,"Erlang Redis Client"},
-              {vsn,"1.3.3"},
+              {vsn,"1.4.0"},
```

Comment:
There is also a `@version` tag in mix.exs which is used when we publish to Hex. Can you update that one too?

Normally we would tag a new version ourselves in a separate commit, but in this case I appreciate that you do it, since we don't have any other features planned. (I think we should add some contributing guidelines to the repo later.)